data/hai/unfettered side missions.txt

```diff
@@ -161,18 +97,18 @@ mission "Hai Legacy 1"
 					decline
 			label first
 			`As you're browsing the shipyard, you are discreetly approached by a hooded figure. When you stop to see what they want to say, they grab and pull your arm to follow them. "Keep walking," the stranger hisses. After pointing at and commenting on random ship, as if to mimic the actions of regular passerby, they address you again. "My associates and I have a lucrative business proposition for you. If you are interested, come find us at these coordinates." Without waiting for a response, they fold a piece of paper into your hand before quickly fading into the crowd. It lists only a location: a beach on the other side of the planet.`
-			`	However, it also mentions you need to have at least 6.5 millions in cash! It may be a good idea to save up or take a debt and then come back another day to head to the location.`
+			`	However, it also mentions you need to have at least 10 millions in cash! It may be a good idea to save up or take a debt and then come back another day to head to the location.`
```

Comment:
```suggestion
			`	However, it also mentions you need to have at least 10 million in cash! It may be a good idea to save up or take a debt and then come back another day to head to the location.`
```